spec/helpers.rb

```diff
@@ -0,0 +1,5 @@
+module Helpers
```

Comment:
Also, here you can define something like 

```
  def json
    JSON.parse(response.body).with_indifferent_access
  end

```
 So in the spec files you can substitute `response.body` for just `json` 

You can make this changes in nexts PRs